src/actions.js

```diff
@@ -478,6 +478,11 @@ export const reloadWalletRequested = () => ({
   type: types.RELOAD_WALLET_REQUESTED,
 });
 
+export const startWalletLock = (payload) => ({
```

Comment:
We already have a `onStartWalletLock` without payload, isn't it confusing to create this method here? Maybe changing the name is better. What's the difference between them?

And the reducer doesn't do anything with this payload.

Reply:
Oops.... I shouldn't have created this action, removed it in c1c790e1867ea76506df7d8e837d9d17a7fcbdf8
 and using the `onStartWalletLock`